### PR TITLE
Tickets/dm 9156

### PR DESCRIPTION
**Pull request by pschella**

### CODE REVIEW

File: python/lsst/obs/subaru/SConscript [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
-scripts.BasicSConscript.pybind11(["crosstalk",
-                                  "hscDistortion"])
+from lsst.sconsUtils import env, targets
 
+# Cannot use here?
```
Comment (jdswinbank) [inline python/lsst/obs/subaru/SConscript]: I dunno if you can use it here or not, but if you can't you don't want it here-but-commented. :-)

File: ups/obs_subaru.cfg [hunk 2]
```diff
@@ -11,5 +11,5 @@ dependencies = {
 config = lsst.sconsUtils.Configuration(
     __file__,
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
+    hasSwigFiles=False,
```
Comment (r-owen) [inline ups/obs_subaru.cfg]: I suggest deleting this line (despite the fact that the current default is True)

File: ups/obs_subaru.table [hunk 3]
```diff
@@ -7,7 +7,7 @@ setupRequired(pyfits)
 setupRequired(sconsUtils)
 setupRequired(pybind11)
 
-setupOptional(meas_modelfit)
+#setupOptional(meas_modelfit)
```
Comment (r-owen) [inline ups/obs_subaru.table]: Please add a short comment explaining why this is commented out and what will be required to restore it (both of which presumably amount to "not yet wrapped with pybind11")

File: ups/obs_subaru.table [hunk 4]
```diff
@@ -7,7 +7,8 @@ setupRequired(pyfits)
 setupRequired(sconsUtils)
 setupRequired(pybind11)
 
-setupOptional(meas_modelfit)
+# temporarily disable meas_modelfit until it is wrapped with pybind11
```
Comment (PaulPrice) [inline ups/obs_subaru.table]: What's the ticket for re-enabling it?

**Pull request merged**